### PR TITLE
Phase I of Volunteer flow

### PR DESCRIPTION
**Pull request by laredotornado**
Related to this

https://github.com/factn/resilience-app/issues/280

There are going to be multiple PRs.  Thi sis the first

### CODE REVIEW

File: src/app/page/Home/VolunteerHome.jsx [hunk 1]
```diff
@@ -2,39 +2,68 @@ import { AppBar, Box, Tab, Tabs, Typography } from "@material-ui/core";
...
+  getAllPlanningMissionss,
   getAllCompletedMissions,
-  getAllStartedMissions,
-  getAllSuggestedMissions,
+  getAllInProgressMissions,
+  getAllProposedMissions,
 } from "./missionHelpers";
 import VolunteerHomeMissionList from "./VolunteerHomeMissionList";
 
+const theme = createMuiTheme({
```
Comment (qdozaq) [inline src/app/page/Home/VolunteerHome.jsx:16]: If you want to steal the styles from here this should prevent you from having to overwrite the theme in this way.
https://github.com/factn/resilience-app/blob/master/src/app/page/Request/foodbox/FoodboxFlow.jsx#L42-L48

And then we can lift this up into it's own component in another pr

**Comment by jwu910** [comment]
Just taking a quick look through this. Are you able to rebase on upstream/master and create a feature branch? I'm seeing 40+ commits and its hard to follow what changes are being proposed here.

File: src/app/page/Home/VolunteerHome.jsx [hunk 2]
```diff
@@ -47,68 +76,78 @@ export default function VolunteerHome({ currentUser }) {
...
+          <Tabs
+            TabIndicatorProps={{ style: { background: "#373985" } }}
+            classes={classes}
+            value={value}
+            variant="fullWidth"
+            centered
+            onChange={handleChange}
+            aria-label="volunteer dashboard tabs"
+          >
+            <Tab label={proposedLabel} {...a11yProps(2)} />
```
Comment (azmainamin) [inline src/app/page/Home/VolunteerHome.jsx:118]: Why are the numbers not in order in allyProps?

**Comment by azmainamin** [comment]
@laredotornado  
image

Can you look at this?

File: src/app/page/Home/VolunteerHome.jsx [hunk 3]
```diff
@@ -47,68 +76,78 @@ export default function VolunteerHome({ currentUser }) {
...
+            <Tab label={planningLabel} {...a11yProps(1)} />
+            <Tab label={inProgressLabel} {...a11yProps(0)} />
+            <Tab label={completedLabel} {...a11yProps(3)} />
+          </Tabs>
+        </AppBar>
+      </ThemeProvider>
       <TabPanel value={value} index={0}>
         <VolunteerHomeMissionList
-          missions={startedMisssions}
+          missions={inProgressMissions}
```
Comment (azmainamin) [inline src/app/page/Home/VolunteerHome.jsx:127]: Can we make sure the TabPanel component mirrors the order of the Tab component? If ProposedTab is at index 0, TabPanel for Proposed Mission should be at 0.

File: src/app/page/Home/VolunteerHome.jsx [hunk 4]
```diff
@@ -2,39 +2,68 @@ import { AppBar, Box, Tab, Tabs, Typography } from "@material-ui/core";
...
+  const [completedMissions, updateCompletedMissions] = useState([]);
 
   useEffect(() => {
     const fetchAllAssociatedMissions = async () => {
       const missions = await User.getAllAssociatedMissions(currentUser.uid);
 
-      updateAssignedMissions(getAllAssignedMissions(missions, currentUser));
-      updateStartedMissions(getAllStartedMissions(missions, currentUser));
-      updateSuggestedMissions(getAllSuggestedMissions(missions, currentUser));
+      updateInPlanningMissions(getAllPlanningMissionss(missions, currentUser));
```
Comment (jwu910) [inline src/app/page/Home/VolunteerHome.jsx:64]: `Missionss` typo seems to be in multiple places, we should fix

**Comment by aamin21** [comment]
@laredotornado  Given there is another PR, should this be closed?

**Comment by jwu910** [comment]
Github showing this is being sent from an unknown repository. @laredotornado If this is still relevant, can you send from your current working branch once you're up to date? Thanks
